src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/telemetry/telemetry.component.ts

```diff
@@ -147,9 +147,11 @@ export class TelemetryComponent implements OnInit {
     this.mgrModuleService.updateConfig('telemetry', config).subscribe(
       () => {
         this.disableModule(
-          'Your settings have been applied successfully. ' +
-            'Due to privacy/legal reasons the Telemetry module is now disabled until you ' +
-            'complete the next step and accept the license.',
+          this.i18n(
+            `Your settings have been applied successfully. \
+Due to privacy/legal reasons the Telemetry module is now disabled until you \
```

Comment:
Wow, the indentation looks... interesting. But I guess there is no way around it?

Reply:
yeah, its starts counting white spaces from the beginning of the line, so we really need to do it this way.
And in the future (Angular 9 I18N) we will need to use just 1 string, without concatenations, so its the only way.

Reply:
Thanks for the explanation!